src/Symfony/Contracts/Service/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=8.0.2",
-        "psr/container": "^1.1"
+        "psr/container": "^1.1|^2.0"
```

Comment:
To  me, this should require `psr/container` 2.0, so that our own interfaces extending the PSR `ContainerInterface` have a fixed signature in `symfony/service-contracts` 3.0 instead of having a signature that depends on the version of the transitive dependency.